quantinuum_schemas/models/backend_config.py

```diff
@@ -224,6 +252,146 @@ class QulacsConfig(BaseBackendConfig):
     result_type: str = "state_vector"
 
 
+class SeleneConfig(BaseModel):
```

Comment:
Is this config also a valid config? If not it might be worth calling it `BaseSeleneConfig` or similar so people don't try and use it. You could even give it an _ prefix.